### PR TITLE
sync with latest approved changes

### PR DESCRIPTION
**Pull request by DiCesareDaniele**
Just some doc fixes

- Deleted some stuff that was removed in the code but not in the docs.
- Fixed/Updated Ack, Nack (assuming #96 gets merged).
- Fixed fragment reassembly section.
- Changed message to packet where they were used interchangeably

### CODE REVIEW

**Review by matteo-dinoia — COMMENTED** [review]
A bit too much in the same PR, it is not very readable, but seem fine.

File: AP-protocol.md [hunk 1]
```diff
@@ -464,45 +428,3 @@ The Simulation Controller can receive the following events from nodes:
...
-	RespFilesList(Vec<u64>),
-	RespFile(Vec<u8>),
-	RespMedia(Vec<u8>),
-	ErrUnsupportedRequestType,
-	ErrRequestedNotFound,
-
-	RespClientList(Vec<NodeId>),
-	RespMessageFrom { from: NodeId, message: Vec<u8> },
-	ErrWrongClientId,
-}
```
[the remark this answers is not in the document]
Comment (DiCesareDaniele) [reply AP-protocol.md:499]: I got you

**Comment by matteo-dinoia** [comment]
It should be clear that this are only a base but the group can decide to do as they wish as long as they follow the main requirement.

**Comment by DiCesareDaniele** [comment]
> It should be clear that this are only a base but the group can decide to do as they wish as long as they follow the requirement in the main-project.pdf

Added a comment at the end of the high level messages section

File: AP-protocol.md [hunk 2]
```diff
@@ -217,41 +219,13 @@ The flood can terminate when:
...
-pub struct MessageData {
-	session_id: u64,
-	content: MessageContent
-}
-```
-
 ### Ack
 
-If a drone receives a Message and can forward it to the next hop, it also sends an Ack to the client.
+This Packet is sent by a client/server after receiving a Packet.
```
Comment (Suge42) [inline AP-protocol.md:228]: Here should be 'Ack' instead of the First 'Packet'

Comment (DiCesareDaniele) [reply AP-protocol.md]: Solved

**Comment by DiCesareDaniele** [comment]
I have also added latest merged commits changes to the protocol

**Comment by DiCesareDaniele** [comment]
If there are other things that needs to be solved let me know

**Comment by matteo-dinoia** [comment]
Merging it to move other PR along

**Pull request merged**